x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityCcrMigrationIT.java

```diff
@@ -162,7 +162,7 @@ public void testRcs1Setup() throws Exception {
 
     // First migrate to RCS 2.0
     @Order(30)
```

Comment:
Note: the test ordering is no longer necessary here since no cluster restarts are required anymore. However, I'd prefer to defer this clean up to keep the PR focused.